src/nvim/tui/tui.c

```diff
@@ -16,6 +16,11 @@
 #include "nvim/os/event.h"
 #include "nvim/tui/tui.h"
 
+// Space reserved in the output buffer to restore the cursor to normal when
```

Comment:
> But you're allocating OUTBUF_SIZE bytes, right? Or is that not what you meant by Space reserved?

`OUTBUF_SIZE` is the total size of the output buffer, "space reserved" is the size in the output buffer reserved for the cnorm command